OpenRA.Mods.Common/UtilityCommands/Glob.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace OpenRA.Mods.Common.UtilityCommands
 {
-	static class Glob
+	public static class Glob
```

Comment:
Wouldnt it make sense to move this too? Its only used by `ConvertPngToShpCommand`